src/styles/_header.scss

```diff
@@ -38,4 +39,21 @@
     font-size: 12px;
     font-weight: 500;
   }
+
+  &__item:hover {
+    color: #34568b;
+    transform: rotate(5deg) translateY(-8px);
+  }
+
+  &__item:hover::after {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    content: "";
+    background-color: #00acdc;
+    height: 4px;
+    width: 100%;
+    border-radius: 8px;
+  }
+
 }
```

Comment:
Use nesting here more details in  [video](https://mate.academy/learn/css-advanced/sass#/video/925)